app/functions.sh

```diff
@@ -8,6 +8,25 @@
 [[ -z "${END_HEADER:-}" ]] && \
  declare -r END_HEADER='## End of configuration add by letsencrypt container'
 
+function check_nginx_proxy_container_run {
+    local _nginx_proxy_container=$(nginx_proxy_container)
+    if [[ $(docker_api "/containers/${_nginx_proxy_container}/json" | jq -r '.State.Status') = "running" ]];then
+        return 0
+    fi
+
+    echo "$(date "+%Y/%m/%d %T"), Error: nginx-proxy container ${_nginx_proxy_container}  isn't running" >&2
```

Comment:
Could you end the error messages with a `.` ? 😃

Reply:
Of course)